statsmodels/nonparametric/kernel_regression.py

```diff
@@ -83,10 +91,14 @@ class KernelReg(GenericKDE):
         The bandwidth parameters.
     """
     def __init__(self, endog, exog, var_type, reg_type='ll', bw='cv_ls',
-                 defaults=None):
+                 ckertype='gaussian', okertype='wangryzin',
+                 ukertype='aitchisonaitken', defaults=None):
         self.var_type = var_type
         self.data_type = var_type
         self.reg_type = reg_type
+        self.ckertype = ckertype
```

Comment:
Can you add a check that these are value kernel names, and raise if not with a meaningful error.

Reply:
Sure done, and added relevant unit test

Reply:
@bashtage Any idea why the build checks below fail now though? Looks like it's failing when collecting tests, but it works fine locally..

Reply:
Never mind, figured it out. Looks like a documented issue with pytest fixtures, that I managed to work around